base/float.jl

```diff
@@ -711,6 +711,74 @@ end
     eps() = eps(Float64)
 end
 
+"""
+    eps(::Type{<:AbstractFloat})
```

Comment:
The `where` was better actually, since as Tony mentioned the parameter `T` is referenced in the explanation.